requirements/common.txt

```diff
@@ -17,6 +17,7 @@ pillow  # Required for image processing
 prometheus-fastapi-instrumentator >= 7.0.0
 tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer >= 0.10.11, < 0.11
+llguidance>=0.6.15
```

Comment:
```suggestion
llguidance>=0.6.15; platform_machine == "x86_64" && platform_machine == "arm64"
```

Reply:
I think it should be "or" not "&&"

Which other architectures would be of interest?